shared/router-v2/tab-bar/container.js

```diff
@@ -16,9 +17,12 @@ const mapStateToProps = state => ({
   username: state.config.username,
 })
 
-const mapDispatchToProps = dispatch => ({
+const mapDispatchToProps = (dispatch, ownProps) => ({
   _onProfileClick: username => dispatch(ProfileGen.createShowUserProfile({username})),
   _onTabClick: tab => {
```

Comment:
does this mean that if you leave the people tab via, say, the chat button on someone's profile, the badge won't be cleared?

Reply:
I think this is the case but don't want to change the UX since I assume this was discussed. @keybase/react-hackers this is ready